src/Components/NavBar/NavBarMobileMenu/NavBarMobileMenuNotificationsIndicator.tsx

```diff
@@ -0,0 +1,82 @@
+import { useContext } from "react"
+import * as React from "react"
+import { graphql } from "relay-runtime"
+import { isServer } from "Server/isServer"
+import { NavBarMobileMenuNotificationsIndicatorQuery } from "__generated__/NavBarMobileMenuNotificationsIndicatorQuery.graphql"
+import { SystemContext } from "System/SystemContext"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { checkAndSyncIndicatorsCount } from "../helpers"
+import { createFragmentContainer } from "react-relay"
+import { NavBarMobileMenuNotificationsIndicator_me } from "__generated__/NavBarMobileMenuNotificationsIndicator_me.graphql"
+import { NavBarNotificationIndicator } from "../NavBarNotificationIndicator"
+
+interface NavBarMobileMenuNotificationsIndicatorProps {
+  me?: NavBarMobileMenuNotificationsIndicator_me | null
+}
+
+export const NavBarMobileMenuNotificationsIndicator: React.FC<NavBarMobileMenuNotificationsIndicatorProps> = ({
+  me,
+}) => {
+  const { hasConversations, hasNotifications } = checkAndSyncIndicatorsCount({
+    notifications: me?.unreadNotificationsCount,
+    conversations: me?.unreadConversationCount,
+  })
+  const shouldDisplayIndicator = hasConversations || hasNotifications
+
+  if (shouldDisplayIndicator) {
+    return (
+      <NavBarNotificationIndicator position="absolute" top="5px" right="5px" />
+    )
+  }
+
+  return null
```

Comment:
**question:** how do you feel about using early return pattern here? something like:

```suggestion
  if (!shouldDisplayIndicator) {
    return null
  }

  return (
      <NavBarNotificationIndicator position="absolute" top="5px" right="5px" />
    )
```